### DISCUSSION TITLE
Hazelcast extension similar to OpenTrace

### DISCUSSION THREAD

**Discussion by harveychong** [open-telemetry/opentelemetry-java-instrumentation#9615] [Q&A]
I would like to know if OpenTelemetry has an extension whose functions similar to `java-hazelcast`(`OpenTrace`)?

**Comment by trask** [comment]
hi @harveychong, I'm not familiar with `java-hazelcast`(`OpenTrace`), can you describe what it does and what you are looking for? thx

**Comment by harveychong** [comment]
`Hazelcast` uses `IExecutorService` extending `ExecutorService`  to submit `Callable` objects to `Hazelcast` servers.  After debugging the code, I found the `executors` extension has ingested `VirtualField` to our `Callable` objects. 
OpenTracing is not required.  Thanks for your help.

FYI: you can find the source of OpenTracing  from https://github.com/opentracing-contrib/java-hazelcast/tree/master
